avaframe/com1DFA/com1DFA.py

```diff
@@ -62,6 +62,9 @@ def initialiseRun(avaDir, flagEnt, flagRes, cfgPar, inputf='shp'):
         os.makedirs(workDir)
     log.info('Directory: %s created' % workDir)
 
+    # Set flag if there is an entrainment or resistance area
+    flagEntRes = False
+
```

Comment:
This logic of when and why something is computed needs to be in a prominent spot of the documentation...

Reply:
I agree!